### PR TITLE
jupyter extract demo + tweaks

### PR DESCRIPTION
**Pull request by cyrush**

### CODE REVIEW

File: src/ascent/ascent_jupyter_bridge/README.md [hunk 1]
```diff
@@ -18,12 +18,24 @@ Bridge Kernel is not on PyPI yet, please clone and install from local source.
 
 spack/bin/spack install -v py-pip ^python@3.6.3
 spack/bin/spack activate py-pip
-pip install ipykernel
+pip install -r requirements.txt
 pip install . [--user] [--upgrade]
```
Comment (cyrush) [inline src/ascent/ascent_jupyter_bridge/README.md]: @SeifIbrahim 
we may want to add a note about how to persevere in the face of SSL cert errors.

`pip install --trusted-host pypi.org --trusted-host files.pythonhosted.org -r requirements.txt`

**Comment by cyrush** [comment]
matplotlib is in the server -- I get why, but I am wondering if there is a clever way to keep it in the client?

**Comment by cyrush** [comment]
Docs are looking really good!

**Comment by cyrush** [comment]
ran the demos, they look great!

**Comment by mclarsen** [comment]
Consider adding text to the changelog.

**Comment by cyrush** [comment]
@mclarsen good suggestion for the changelog
@SeifIbrahim can you add an entry (https://github.com/Alpine-DAV/ascent/blob/develop/CHANGELOG.md) and then I think we ready to merge, unless you have any more final tweaks.

**Comment by cyrush** [comment]
@mclarsen I'll tackle the changelog in another PR

**Pull request merged**